ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent-prod.json

```diff
@@ -12,5 +12,19 @@
     "CaseEventID": "draftCMO",
     "UserRole": "[LASOLICITOR]",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "actionCMO",
+    "UserRole": "caseworker-publiclaw-judiciary",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
Do we need to give admin read permissions for event also to see it in tab?

Reply:
This is the old event so I am not bothered with it (it will be removed when we toggle on); admin have read permissions for the new event + the case field that will show in the tab

Reply:
Ah ofc, makes sense then. :)